docs/roleplay/mastery.md

```diff
@@ -0,0 +1,135 @@
+# Bending Mastery
+
+## Becoming a Master
+
+For a character to become a master of their element they must complete an average of 10 roleplay lessons, minimum 8, depending on their current skill level, age, and ability. A minimum of 8/10 lessons must be completed on the server. 2/10 can be in the [Roleplay Hub](https://discord.gg/tjqX25pH37) if necessary.
+
+## Bending Instructors
+
+### Types of Instructor
+
+There are two types of bending instructor:
+
+- **Masters** have completed all lessons and have an accepted mastery application
+- **Teachers** have completed any amount of the first 5 lessons and can pass their knowledge on to others
+
+### Finding an Instructor
+
+Players are encouraged to find bending instructors through roleplay. However, if your character cannot find one, you can look for a player with the `Instructor` tag in the Roleplay Hub and contact them about training. Bending teachers and masters can choose to have this tag if they would like to be contacted by others. You can then arrange to have your characters meet in a realistic way.
+
+### Role of Instructors
+
+- Follow the lesson structure when teaching and determine if a character will need more or less training based on skill level and progress shown. The structure is flexible depending on teaching style and the student's interests. 
+- Use the `#instructors` channel in the Roleplay Hub to share training ideas, more detailed guides to specific lessons, and communicate students' progress.
+- When a student applies for mastery, reply to the application to confirm which lessons you have taught them and if you think they are ready to be a master.
+
+## Lesson Structure
+
+The first 5 lessons can be completed with a master *or* teacher *or* mix of both. The last 5 lessons must be completed with a master. [Water](https://avatar.fandom.com/wiki/Waterbending#Waterbending_master_level), [Earth](https://avatar.fandom.com/wiki/Earthbending#Earthbending_master_level), and [Fire](https://avatar.fandom.com/wiki/Firebending#Firebending_master_level) have moves classed as master level. Students can choose to learn two of these as part of their mastery training. Along with the outlines below, lessons can include sparring, training with other students, or visiting locations in order to learn a specific skill. Be creative!
+
+### Water
+Waterbending students can choose to follow either the traditional, or healing path to mastery. If choosing the healing path, offensive lessons will be replaced by healing lessons. For practical waterbending or history and culture, the lesson can relate specifically to the character's Clan if applicable.
```

Comment:
I had to read this sentence twice to understand, I stumbled over the comma. I think removing the comma makes it clearer?
```suggestion
Waterbending students can choose to follow either the traditional or healing path to mastery. If choosing the healing path, offensive lessons will be replaced by healing lessons. For practical waterbending or history and culture, the lesson can relate specifically to the character's Clan if applicable.
```